tests/movie.model.test.js

```diff
@@ -0,0 +1,80 @@
+const service = require("../services/user.service");
```

Comment:
It'll be good if you include tests for the corner cases as well and not just the stright scenarios. What is happeining when you try to delete a movie which does not exists, ot the store is empty, when you search for movie which does not exists, etc.
You can also stub the service `writeJSONFile` method to throw error and test that it's properly handled.